pex/resolve/locked_resolve.py

```diff
@@ -282,3 +430,203 @@ def emit_artifact(
                     additional_artifact,
                     line_continuation=index != len(locked_requirement.additional_artifacts),
                 )
+
+    def resolve(
+        self,
+        target,  # type: Target
+        requirements,  # type: Iterable[Requirement]
+        transitive=True,  # type: bool
+        build=True,  # type: bool
+        use_wheel=True,  # type: bool
+        prefer_older_binary=False,  # type: bool
+    ):
+        # type: (...) -> Union[Resolved, Error]
+
+        is_local_interpreter = isinstance(target, LocalInterpreter)
+        if not use_wheel:
+            if not build:
+                return Error(
+                    "Cannot both ignore wheels (use_wheel=False) and refrain from building "
+                    "distributions (build=False)."
```

Comment:
How challenging would it be to mention the lockfile's path in this error message? I'm thinking something like:

> Cannot resolve lockfile at /path/to/lock.txt: Cannot both ignore wheels ...

This isn't as relevant when using PEX directly via CLI, but would help when using via Pants.

Reply:
This is easy enough to do - I'll follow up this PR with a smaller PR to plumb this.